### PR TITLE
Feature#39/login view

### LINKED ISSUE #38: create login view
# REQUIREMENTS
Create the login views adapting to the desktop and mobile design.

# DESIGN
image

# Acceptance Criteria
- [x] Mobile and desktop view
- [x] Funtional sign in

### PR DESCRIPTION
**Pull request by jairolan**
# Description
What did you implemented/Why did you implemented this:
 - I added a specific scss file for the user login view.
 - I did that so as not to mix the css classes directly in the application.css file.
 - I updated the development.rb file. 
 - That was done so that everyone on the team can log in as an administrator.
 - The login view was designed in its mobile and desktop version. 
   
  ## Screenshots
- Desktop view
image
- Mobile view
image

### CODE REVIEW

**Review by oramasangel — DISMISSED** [review]
I like the design for desktop

**Review by xFerchoVJ — DISMISSED** [review]
LGTM!

File: reloj-checador/app/javascript/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+@import url(devise.scss);
```
Comment (juliocabrera820) [inline reloj-checador/app/javascript/stylesheets/application.scss]: I think you can import the scss file in another way, @Chavin30 is there another way?

File: reloj-checador/app/javascript/stylesheets/devise.scss [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+  text-align: center;
+}
+
+.centered-form{
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  text-align: center;
+}
```
Comment (juliocabrera820) [inline reloj-checador/app/javascript/stylesheets/devise.scss]: It looks like duplicated code

File: reloj-checador/app/views/devise/sessions/new.html.erb [hunk 3]
```diff
@@ -1,24 +1,45 @@
...
+  <div class="split left">
+	<h2>Welcome to the check clock</h2>
+		<div class="centered-form">
+		
+			<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>	
+				<div class="field">
+					
+					<%= f.email_field :email, autofocus: true, autocomplete: "email",class:"in-email"%>
+					<p>Email</p>
+				</div>
```
Comment (juliocabrera820) [inline reloj-checador/app/views/devise/sessions/new.html.erb]: keep your indentation and blank lines

File: reloj-checador/app/views/devise/sessions/new.html.erb [hunk 4]
```diff
@@ -1,24 +1,45 @@
...
+					<%= f.password_field :password, autocomplete: "current-password", class:"in-password"%>
+					<p>Password</p>
+				</div>
+
+				<% if devise_mapping.rememberable? %>
+					<div class="field">
+						<%= f.check_box :remember_me %>
+						<%= f.label :remember_me %>
+					</div>
+				<% end %>
```
Comment (juliocabrera820) [inline reloj-checador/app/views/devise/sessions/new.html.erb]: keep your indentation and blank lines

File: reloj-checador/app/views/devise/sessions/new.html.erb [hunk 5]
```diff
@@ -1,24 +1,45 @@
...
-  <div class="field">
-    <%= f.label :password %><br />
-    <%= f.password_field :password, autocomplete: "current-password" %>
+				<div>
+					<br>
+				
+					<button class="button-log">Log in <% f.submit %></button>
+				</div>
+			<% end %>
+		</div>
```
Comment (juliocabrera820) [inline reloj-checador/app/views/devise/sessions/new.html.erb]: keep your indendation and blank lines

File: reloj-checador/app/views/devise/sessions/new.html.erb [hunk 6]
```diff
@@ -1,24 +1,45 @@
...
-      <%= f.check_box :remember_me %>
-      <%= f.label :remember_me %>
+  <div class="split right">
+    <div class="centered">
+      <%# <img src="" alt="Avatar man"> %>
+			<a href="/"> 
+        <%= image_tag 'check-login.jpg'%>
+      </a>
+      <%# <h2>John Doe</h2>
+      <p>Some text here too.</p> %>
```
Comment (juliocabrera820) [inline reloj-checador/app/views/devise/sessions/new.html.erb]: commented-out code must be removed

File: reloj-checador/db/seeds/development.rb [hunk 7]
```diff
@@ -1 +1,3 @@
-User.create({ name: 'george', email: 'geo@gmail.com', password: '123456' })
+Company.create([{ name: 'bcoders', address: 'c 34 #850 x 90 y 92' }, { name: 'wise', address: 'c 41 #652 x 70 y 72' },
+                { name: 'line', address: 'c 28 #356 x 21 y 23' }])
+User.create name: 'george', email: 'geo@gmail.com', password: '123456', company_id: 1
```
Comment (juliocabrera820) [inline reloj-checador/db/seeds/development.rb]: This code has been added in another branch, so you can remove it

**Review by juliocabrera820 — CHANGES_REQUESTED** [review]
Nice job ⚡

**Review by oramasangel — DISMISSED** [review]

**Review by oramasangel — APPROVED** [review]

**Review by juliocabrera820 — APPROVED** [review]
LGTM

**Pull request merged**